pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1170,6 +1170,26 @@ func TestSchedulerCache_UpdateNodeInfoSnapshot(t *testing.T) {
 		}
 		pods = append(pods, pod)
 	}
+
+	// Add three with affinity pods, two pods on the first node and one on the second node.
+	for i := 0; i < 3; i++ {
+		id := len(pods) + i
+		pod := &v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      fmt.Sprintf("test-pod%v", id),
+				Namespace: "test-ns",
+				UID:       types.UID(fmt.Sprintf("test-puid%v", id)),
+			},
+			Spec: v1.PodSpec{
+				NodeName: fmt.Sprintf("test-node%v", i%2),
```

Comment:
It's easier to read if we make the `addPod` function like `func (podIndex, nodeIndex int)`. 
Just a suggestion.

Reply:
we don't really need to do this hack, I simplified. Refactoring into a function is not trivial for this test.